dns_sd_avahi.c

```diff
@@ -83,7 +86,13 @@ static void avahi_process_resolved(struct dns_sd_discovery_data *ddata,
 	}
 	iio_mutex_unlock(ddata->lock);
 
-	IIO_DEBUG("\t\t%s:%u (%s)\n", host_name, port, ddata->addr_str);
+	ptr = ddata->addr_str + strlen(ddata->addr_str);
+
+	if (addr->proto == AVAHI_PROTO_INET6
+	    && iface != AVAHI_IF_UNSPEC
+	    && if_indextoname((unsigned int)iface, ptr + 1)) {
+		*ptr = '%';
+	}
```

Comment:
shouldn't we move `IIO_DEBUG` here? Or at least state in the message that you want to explicitly remove that log

Reply:
I actually removed it accidentally. Good catch.